web/src/TabNav.scss

```diff
@@ -24,6 +24,14 @@
   border-left: 1px solid $color-gray-light;
 }
 
+.TabNav span#TabNav-container {
```

Comment:
this...doesn't seem like a good idea. can you elaborate a bit on what you're trying to accomplish here?

Reply:
(we don't really use IDs in our UI code, or floats...there are good reasons to use them but they aren't everyday tools that I would expect someone to need...also happy to pair on this)

Reply:
Ah! yeah i have no problem using a class -- it just seemed weird to me to have a one-time object be a class instead of an id.

I'm trying to put the tabs at one end of this div and the sail url at another. i'd believe there's another way to do this, i just don't know it, so would love to pair